pkg/notes/document/document.go

```diff
@@ -326,70 +322,49 @@ func sortKinds(notesByKind NotesByKind) []Kind {
 	return res
 }
 
+// urlPrefixForBucket returns the URL prefix for the provided bucket string
+func urlPrefixForBucket(bucket string) string {
+	urlPrefix := fmt.Sprintf("https://storage.googleapis.com/%s/release", bucket)
+	if bucket == "kubernetes-release" {
+		urlPrefix = "https://dl.k8s.io"
```

Comment:
Make this a constant in the release pkg.

Reply:
Done